handlers/rest/cluster.go

```diff
@@ -131,7 +131,7 @@ func (crh *clusterRestHandler) Configure(c *gin.Context) {
 		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": err.Error()})
 	}
 
-	err := crh.nodeMgr.InitAppDHT(cf.SlotPerNodeCount)
+	err := crh.nodeMgr.InitAppDHT(cf.Shards, cf.Replicas)
```

Comment:
We are initialising DHT on this endpoint call only, what if we lost a node?

Reply:
Incase of loss of a node, we will fetch the DHT from nodes that are already alive. Via raft FSM. This API call is only for the first time initialisation.